exporter/metric/dogstatsd/dogstatsd.go

```diff
@@ -75,7 +75,7 @@ func NewRawExporter(config Config) (*Exporter, error) {
 // 	defer pipeline.Stop()
 // 	... Done
 func InstallNewPipeline(config Config) (*push.Controller, error) {
-	controller, err := NewExportPipeline(config)
+	controller, err := NewExportPipeline(config, time.Hour)
```

Comment:
this default needs to be on order 1 minute, I'm not sure why we defaulted to 1 hour below...